generators/server/templates/build.gradle.ejs

```diff
@@ -18,7 +18,6 @@
 -%>
 buildscript {
     repositories {
-        mavenLocal()
         mavenCentral()
         gradlePluginPortal()
         maven { url "https://repo.spring.io/plugins-release" }
```

Comment:
Is this line needed? It looks kinda funny to me.

Reply:
Need check. Maybe we can remove all additional buildscript repositories.

Reply:
Removed all repositories except the gradle plugin portal.